react/ProductName.tsx

```diff
@@ -102,6 +109,37 @@ function ProductName({
     )
   }
 
+  if (showProductLink && productLink) {
+    return (
+      <Wrapper
+        className={`${handles.productNameContainer} mv0 ${className ?? ''}`}
+      >
+        <a
```

Comment:
I don't think this is the better way of doing it. We usually use `render-runtime`'s `Link` component to redirect users to the product page. Take a look at this [example](https://github.com/vtex-apps/product-summary/blob/46ade16bc8715835b39281cacec32f4823179b87/react/ProductSummaryCustom.tsx#L140).

I also think we shouldn't duplicate all the code in case there's a link wrap or not. You should create something like:

```suggestion
        const LinkWrapper = showProductLink ? Link : Fragment
        
        // And then use it where you put the a tag previously
        <LinkWrapper {...linkProps}>
          // Rest of the code
        </LinkWrapper>
```

Reply:
I fixed it, following the requested changes. You can find all changes in the "required changes" commit.